docs/set.js

```diff
@@ -0,0 +1,74 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+/**
+ * Instances of this class will be returned when accessing object properties whose type is `"Set"`
+ * (see {@linkplain Realm~ObjectSchemaProperty ObjectSchemaProperty}).
+ *
+
+ * Sets mostly behave like normal JavaScript Sets, with a few exceptions:
+ * They can only store values of a single type (indicated by the `type` 
+ * and `optional` properties of the Set).
+ * They can only be modified inside a {@linkplain Realm#write write} transaction.
+ * Unlike JavaScript's Set, Realm~Set does NOT make any guarantees about the 
+ * traversal order of `values()`, `entries()`, `keys()`, or `forEach` iterations.
+ *
```

Comment:
Perhaps add a note, that if the developer wishes order to be established, they must implement it themselves.

Reply:
Done.